Singularity/Singularity/Platform/PlatformBlank.cs

```diff
@@ -12,9 +13,14 @@
 namespace Singularity.Platform
 {
     [DataContract()]
-    public class PlatformBlank : IDraw, IUpdate, ICollider, IRevealing
+    public class PlatformBlank : ISpatial, IRevealing, INode
```

Comment:
Warum sind die anderen interfaces weg?

Reply:
ISpatial besitzt bereits IDraw, IUpdate und ICollider is wohl beim mergen verloren gegangen ... :D